django/shortcuts.py

```diff
@@ -24,38 +25,59 @@
 from django.utils.functional import Promise
 
 
-def render_to_response(template_name, context=None,
-                       context_instance=_context_instance_undefined,
-                       content_type=None, status=None, dirs=_dirs_undefined,
-                       dictionary=_dictionary_undefined, using=None):
+def _http_response_with(
```

Comment:
It probably makes sense to remove the helper methods that are now used only once.